test/e2e/network/service.go

```diff
@@ -1123,6 +1228,50 @@ var _ = SIGDescribe("Services", func() {
 		framework.ExpectNoError(verifyServeHostnameServiceUp(cs, ns, host, podNames2, svc2IP, servicePort))
 	})
 
+	ginkgo.It("should be able to preserve UDP traffic when server pod cycles [Slow]", func() {
+		serviceName := "clusterip-test"
+		serverPod1Name := "server-1"
+		serverPod2Name := "server-2"
+
+		ns := f.Namespace.Name
+
+		nodeIP, err := e2enode.PickIP(cs) // for later
+		framework.ExpectNoError(err)
+
```

Comment:
`ginkgo.By("creating a UDP service " + serviceName + " with type=NodePort in namespace " + ns)`

Reply:
done